cli/runtime/apis/config/v1alpha1/clientconfig_types.go

```diff
@@ -160,6 +160,7 @@ type CLIOptions struct {
 	DiscoverySources []PluginDiscovery `json:"discoverySources,omitempty" yaml:"discoverySources"`
 	// UnstableVersionSelector determined which version tags are allowed
 	UnstableVersionSelector VersionSelectorLevel `json:"unstableVersionSelector,omitempty" yaml:"unstableVersionSelector"`
+	// Deprecated: BOMRepo has been deprecated and will be removed from future version
```

Comment:
I don't think this belongs here. It is duplicated on line 166 where it actually belongs

Reply:
Thanks for catching this. Updated the PR.